OpenRA.Game/Support/Log.cs

```diff
@@ -75,22 +77,39 @@ public static void AddChannel(string channelName, string baseFilename)
 			}
 		}
 
-		public static void Write(string channel, string value)
+		public static void Write(string channelName, string value)
 		{
-			var writer = Channel(channel).Writer;
+			var channel = Channels[channelName];
+			var writer = channel.Writer;
 			if (writer == null)
 				return;
 
-			writer.WriteLine(value);
+			if (!channel.IsTimestamped)
```

Comment:
AFAIR general code style is to NOT have braces when there is only one line in a block.

Reply:
If this is meant to be enforced, it should probably be specified in the StyleCop config file. I've gone ahead a made the changes.

Reply:
There is no StyleCop setting to adhere. StyleCop does not have a rule for single-line-statements-having-braces.

Reply:
Yikes!